agent/api/task.go

```diff
@@ -251,10 +255,14 @@ func (task *Task) addNetworkResourceProvisioningDependency(cfg *config.Config) {
 		if container.IsInternal() {
 			continue
 		}
-		if container.SteadyStateDependencies == nil {
-			container.SteadyStateDependencies = make([]string, 0)
+		if container.TransitionDependencySet.ContainerDependencies == nil {
+			container.TransitionDependencySet.ContainerDependencies = make([]ContainerDependency, 0)
 		}
-		container.SteadyStateDependencies = append(container.SteadyStateDependencies, PauseContainerName)
+		container.TransitionDependencySet.ContainerDependencies = append(container.TransitionDependencySet.ContainerDependencies, ContainerDependency{
+			ContainerName:   PauseContainerName,
+			SatisfiedStatus: ContainerRunning,
```

Comment:
This should be `ContainerResourcesReady`

Reply:
Done